tendrl/commons/flows/create_cluster/ceph_help.py

```diff
@@ -161,6 +163,40 @@ def create_osds(parameters, created_mons):
 
             wait_for_task(task_id)
 
+            journal_details = {}
+            try:
+                journal_details = json.loads(NS.etcd_orm.client.read(
+                    'nodes/%s/journal_details' % node
+                ).value.decode('utf-8'))
+            except etcd.EtcdKeyNotFound:
+                pass
+
+            if config["journal_colocation"]:
+                for entry in devices:
+                    journal_details[entry] = {
+                        'journal_count': 1,
+                        'ssd': False,
+                        'journal_size': config['journal_size'] * 1024 * 1024
+                    }
+            else:
+                for k, v in devices.iteritems():
+                    journal_disk_name = v
+                    if journal_disk_name in journal_details.keys():
+                        journal_details[journal_disk_name]['journal_count'] += 1
+                        journal_details[journal_disk_name]['ssd'] = True
+                    else:
+                        journal_details[journal_disk_name] = {
+                            'journal_count': 1,
+                            'ssd': False,
+                            'journal_size': config['journal_size'] * 1024 * 1024
+                        }
+
+            NS.etcd_orm.client.write(
```

Comment:
- Avoid using the raw client writes, the reasons for this have been widely discussed in the past
- This is sds specific data and belongs to /clusters/$integration_id/ via a new object called integrations.ceph.objects.Journal (attrs: osd_uuid [0]... other journal attributes)
- also create integrations.ceph.objects.Osd, instances of which are created during ceph osd create.

[0]: http://ceph.com/geen-categorie/ceph-osd-uuid-conversion-to-osd-id-and-vice-versa/